exercises/diamond/example.js

```diff
@@ -5,7 +5,7 @@ var Diamond = function () {
     for (var i = 0; i <= inputIndex; i++) {
       output += getLine(inputIndex, i);
     }
-    for (var i = inputIndex - 1; i >= 0; i--) {
+    for (i = inputIndex - 1; i >= 0; i--) {
```

Comment:
In this case, `i` is used in two different loops. I prefer not using the same variable to control two or more different loops, just in case I don't pay attention and I forget to initialize it in the second loop.

Instead of removing the second declaration, what about using a different name in the second loop? would it make the solution more readable?